demo/atom/textarea/playground

```diff
@@ -0,0 +1,100 @@
+return (
+  <div>
+    <h1>AtomTextarea</h1>
+    <h2>Basic Usage</h2>
+    <p>
+      <code>short</code> <em>size</em> by default
+    </p>
+    <div
+      style={{
+        border: '1px solid #CCC',
+        background: '#fff',
+        marginTop: '10px',
+        padding: '10px'
+      }}
+    >
+      <AtomTextarea label="Description" />
+    </div>
+    <h2>Long size</h2>
+    <div
+      style={{
+        border: '1px solid #CCC',
+        background: '#fff',
+        marginTop: '10px',
+        padding: '10px'
+      }}
+    >
+      <AtomTextarea label="Description" size="long">Saul Bass on failure: Failure is built into creativity</AtomTextarea>
+    </div>
+    <h2>Basic Usage w/ ID</h2>
+    <div
+      style={{
+        border: '1px solid #CCC',
+        background: '#fff',
+        marginTop: '10px',
+        padding: '10px'
+      }}
+    >
+      <AtomTextarea  label="Description" id="hola-amigos">Saul Bass on failure: Failure is built into creativity</AtomTextarea>
+    </div>
+    <h2>Setting Max Characters size</h2>
+    <div
+      style={{
+        border: '1px solid #CCC',
+        background: '#fff',
+        marginTop: '10px',
+        padding: '10px'
+      }}
+    >
+      <AtomTextarea label="Description"  maxCharacters={100}>Saul Bass on failure: Failure is built into creativity</AtomTextarea>
+    </div>
+    <h2>Disabled</h2>
+    <div
+      style={{
+        border: '1px solid #CCC',
+        background: '#fff',
+        marginTop: '10px',
+        padding: '10px'
+      }}
+    >
+      <AtomTextarea  label="Description" disabled>Saul Bass on failure: Failure is built into creativity</AtomTextarea>
+    </div>
+    <h2>Success</h2>
+    <div
+      style={{
+        border: '1px solid #CCC',
+        background: '#fff',
+        marginTop: '10px',
+        padding: '10px'
+      }}
+    >
+      <AtomTextarea  label="Description" success successText="Perfect!">
+        Saul Bass on failure: Failure is built into creativity
+      </AtomTextarea>
+    </div>
+    <h2>Error</h2>
+    <div
+      style={{
```

Comment:
To make your demo more readable, you should store these common styles in a const